lib/jsdom/browser/Window.js

```diff
@@ -556,6 +556,97 @@ function Window(options) {
 
     forEachSheetRuleOfElement(elt, setPropertiesFromRule);
 
+    /**
+     * properties for which `getResolvedValue` is implemented which is less than
+     * every supported property according to step 5
+     * @type Record<string, { inherited: boolean; initial: unknown; computedValue: 'as-specified' }>
```

Comment:
Can you remove the jsdoc/type annotations and just have this be a normal `//` comment? Here and below.

Reply:
done